src/Chapter1.hs

```diff
@@ -636,7 +650,8 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = read [head $ show n]
```

Comment:
Very creative solution! :smile: :art:

Reply:
Thank you for all the feedback so far! Haskell is very... daunting, but I love learning what I can about it!